subprojects/packagefiles/libgpiod/man/meson.build

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+if get_option('manpages')
+    help2man_prog = find_program('help2man')
+    tools = [ gpiodetect, gpioinfo, gpioget, gpioset, gpiofind, gpiomon ]
+    mandir = join_paths(get_option('prefix'), get_option('mandir'))
+    man1 = join_paths(mandir, 'man1')
+    manpages = []
+    foreach t : tools
+        custom_target('@0@.1'.format(t.name()),
```

Comment:
For the record, you don't need t.name() here, you could go the other direction and use:

```meson
tools = [ 'gpiodetect', 'gpioinfo', 'gpioget', 'gpioset', 'gpiofind', 'gpiomon' ]
foreach t: tools
    toolprog = get_variable(t)
    custom_target(...)
```

Reply:
Hah, I wasn't aware of `get_variable()`. This works for me.